minvws/nl-kat-coordination
# Make the "name" field for plugins mandatory

### Changes

A follow up on https://github.com/minvws/nl-kat-coordination/pull/3444 where we make the `name` field required at the model level as well.

### Issue link

This should fix the issue with the database models where the name is not nullable, and make sure we can use the name field for human readables now that we will start generating UUIDs for plugins ids. This PR also adds some names that were still missing.

### Demo

-

### QA notes

This does not add features, please verify that the boefjes and normalizers run properly still!

---

### Code Checklist




---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
